src/react/databrowser/objects/ObjectListTable.jsx

```diff
@@ -134,23 +152,36 @@ export default function ObjectListTable({ objects, bucketName, toggled, isVersio
         initialState: { hiddenColumns },
     }, useFilters, useSortBy, useFlexLayout);
 
+    const refList = useCallback((ref) => {
```

Comment:
Here we are assuming that scrollbar will be 8px wide and so that it will move our cells 8px to the left.
That seems to be true in Chrome but Firefox scrollbar is not taking any width and so we end up with unaligned header and content. 

<img width="1792" alt="Screen Shot 2021-07-01 at 10 47 28 AM" src="https://user-images.githubusercontent.com/7453636/124145470-d4617280-da5a-11eb-8036-21653ae88bf7.png">

Reply:
It seems it is only on MacOS.
I also noticed that on my side firefox and google chrome scrollbar size has a tiny difference.

![image](https://user-images.githubusercontent.com/44285344/124147855-5d929c80-da8f-11eb-8a31-1965ee7d7b15.png)